cmake/modules/jsoncpp.cmake

```diff
@@ -18,6 +18,7 @@ else()
 	set(JSONCPP_INCLUDE "${JSONCPP_SRC}")
 	set(JSONCPP_LIB_SRC "${JSONCPP_SRC}/jsoncpp.cpp")
 	message(STATUS "Using bundled jsoncpp in '${JSONCPP_SRC}'")
+	install(DIRECTORY "${JSONCPP_INCLUDE}/" DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/${LIBS_PACKAGE_NAME}/")
```

Comment:
Why not using same fix as other cmake modules here, ie: adding the trailing slash to `set(JSONCPP_INCLUDE ...` line, for coherence?